config/notifiers.go

```diff
@@ -156,6 +165,24 @@ func (nc *NotifierConfig) SendResolved() bool {
 	return nc.VSendResolved
 }
 
+// DiscordConfig configures notifications via Discord.
+type DiscordConfig struct {
+	NotifierConfig `yaml:",inline" json:",inline"`
+
+	HTTPConfig *commoncfg.HTTPClientConfig `yaml:"http_config,omitempty" json:"http_config,omitempty"`
+	APIURL     *SecretURL                  `yaml:"api_url,omitempty" json:"api_url,omitempty"`
```

Comment:
```suggestion
	WebhookURL     *SecretURL                  `yaml:"webhook_url,omitempty" json:"webhook_url,omitempty"`
```

Discord has several APIs, this uses the webhook API - how do you feel about making it a bit clear?

Reply:
Yes. Sounds good to me!